src/Compiler/RefC/RefC.idr

```diff
@@ -689,7 +640,7 @@ mutual
         pure $ MkRS opStatement opStatement
     cStatementsFromANF (AExtPrim fc _ p args) _ = do
         emit fc $ "// call to external primitive " ++ cName p
-        let returnLine = (cCleanString (show (toPrim p)) ++ "("++ showSep ", " (map varName args) ++")")
+        let returnLine = "idris2_" ++ (cName p) ++ "("++ showSep ", " (map varName args) ++")"
```

Comment:
Here we are losing the logic of a hard fail in case the primitive is not supported.
We should at least keep the check in.